### DISCUSSION TITLE
Room.Leave() 오류창 질문

### DISCUSSION THREAD

**Discussion by jun5640** [naverz/zepeto-studio-kor#361] [5. Q&A]
현재 뒤로가기를 통한 방 나가기를 구현한 상태입니다.
해당 코드는 아래와 같습니다.

 this.go_ClientCode.GetComponent<ClientCode>().GetRoom().Leave(false);

![image](https://user-images.githubusercontent.com/13131384/153359595-894ac3b9-02c8-4317-b4be-3c0335ee068c.png)
퍼블리싱을 진행한 후에  해당 함수를 통해 종료를 하면 아래와 같은 오류창이 나타납니다.

어떤 부분이 문제가 있는지 궁금합니다.

**Comment by runasy-koonta** [comment]
1006 에러는 Websocket 에러 코드인데, 비정상적으로 연결이 끊겼음을 나타내는 오류이며, 별다른 오류 상세사항은 알 수가 없습니다.
다만 Room.Leave()를 사용하신 점을 보면, room을 나갔을 때 multiplay server와의 연결이 종료되니 당연히 Websocket connection이 close되는게 맞을 것 같아 보이고, 정상적인 에러 메시지로 보입니다.
혹시 룸을 나간 뒤 커스텀 에러 메시지를 띄우시고 싶으신건가요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

Leave 함수의 매개변수로 true 값을 사용해서 진행 부탁드립니다.

현재 디폴트 값이 true로 되어있어서  Leave()  이런 식으로 호출해도 괜찮습니다.

추가로 문제가 발생한다면 문의 부탁드립니다.
